pallets/staking/src/lib.rs

```diff
@@ -34,7 +34,10 @@ pub mod weights;
 
 use frame_support::pallet;
 pub(crate) mod hooks;
-mod migrations;
+
+// TODO:: Take it part of PR621
```

Comment:
Is this `TODO` still outstanding?

Reply:
Yup, has to be taken care part of #621

Reply:
Is it possible that we don't make any changes to this file as part of this PR?

Reply:
No, right now not possible, mainly due to the requirements of moving validatorsets from pallet members to invulnerables storage.